package.json

```diff
@@ -13,8 +13,16 @@
     "lib"
   ],
   "exports": {
-    "import": "./lib/esm/index.browser.js",
-    "default": "./lib/cjs/index.js"
+    ".": {
+      "types": "./lib/types/index.d.ts",
+      "browser": "./lib/esm/index.browser.js",
+      "import": "./lib/esm/index.js",
+      "require": "./lib/cjs/index.js"
+    },
+    "./browser": {
```

Comment:
What is the purpose of adding a new export here?

Do bundlers not properly handle the `browser` key on line 18?

Reply:
This is a fallback. If someone isn't using a bundler (or one that supports `browser`), they can also import this by name (`uid-promise/browser`).